docs/user-guide/src/version_support.md

```diff
@@ -53,4 +53,27 @@ Supported container images for BMO and Ironic can be found in quay. Examples are
 Supported container images for CAPM3 and IPAM will always follow the supported release version tags and can be found in quay. Examples are:
 
 - quay.io/metal3-io/cluster-api-provider-metal3:v1.2.0
-- quay.io/metal3-io/ip-address-manager:v1.2.0
\ No newline at end of file
+- quay.io/metal3-io/ip-address-manager:v1.2.0
+
+## CI Test Matrix
+
+The table describes which branches/image-tags are tested in each periodic CI tests:
+
+<!-- markdownlint-disable -->
+
+| TESTS                                                | CAPM3 branch | IPAM branch  | BMO branch/tag  | Keepalived tag | MariaDB | VBMC   | sushi-tools | Ironic |
+| ------------------------------------                 | ------------ | -----------  | --------------- | -------------- | ------- | ------ | ----------- | ------ |
```

Comment:
While we may want to eventually version Ironic and MariaDB, I'd not envision versioning vbmc and sushy-tools (you have a typo in the name btw).

Reply:
Thank for reviewing. I addressed your comments,  Removed sushy-tools and VBMC from the table, added line mentioning we use latest versions of them.